# HG changeset patch
# User Max Horn <max@quendi.de>
# Date 1395263557 -3600
# Branch stable-4.7
# Node ID 070b4a964402c4396625dbf81e8f1a4e830259da
# Parent  388dbf0c64b27825cf19ea9b4f9865a94bb7d561
build: do not use -no_pie linker option on older OS X versions

diff --git a/cnf/aclocal.m4 b/cnf/aclocal.m4
--- a/cnf/aclocal.m4
+++ b/cnf/aclocal.m4
@@ -158,7 +158,9 @@
 AC_DEFUN(GP_LDFLAGS,
 [AC_CACHE_CHECK(Linker default flags, gp_cv_ldflags,
  [ case "$host-$CC" in
-    *-apple-* )
+    *-darwin[[6-9]].* | *-darwin10.* )
+        gp_cv_ldflags="-g ${ABI_CFLAGS}";;
+    *-darwin* )
         gp_cv_ldflags="-g -Wl,-no_pie ${ABI_CFLAGS}";;
     *-gcc* | *-egcs )
         gp_cv_ldflags="-g ${ABI_CFLAGS}";;
diff --git a/cnf/configure.out b/cnf/configure.out
--- a/cnf/configure.out
+++ b/cnf/configure.out
@@ -4435,7 +4435,9 @@
   $as_echo_n "(cached) " >&6
 else
    case "$host-$CC" in
-    *-apple-* )
+    *-darwin[6-9].* | *-darwin10.* )
+        gp_cv_ldflags="-g ${ABI_CFLAGS}";;
+    *-darwin* )
         gp_cv_ldflags="-g -Wl,-no_pie ${ABI_CFLAGS}";;
     *-gcc* | *-egcs )
         gp_cv_ldflags="-g ${ABI_CFLAGS}";;
